dapr-spring/dapr-spring-cloudconfig/src/test/resources/application.yaml

```diff
@@ -0,0 +1,21 @@
+spring:
+  application:
+    name: producer-app
+  config:
+    import:
+      - dapr:secret:democonfig/multivalue-properties?type=doc
+      - dapr:secret:democonfig/dapr.spring.democonfigsecret.singlevalue?type=value
+      #- dapr:secret:democonfigmulti/value1?type=value
```

Comment:
@lony2003 we don't need to keep the comments.

Reply:
Removed.